source/2_platform/rex_directx/include/rex_directx/directx_feature_level.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "rex_engine/types.h"
+#include "rex_std/string.h"
+
+struct IDXGIAdapter;
+
+namespace rex
+{
+  struct FeatureLevelInfo
+  {
+    s32 level;
```

Comment:
this seams to be storing the int version of the enum value.
I don't think this is necessary and we have enum reflection to get the name.
so I think this struct can be removed all together